graphdatascience/query_runner/progress/query_progress_logger.py

```diff
@@ -65,14 +65,27 @@ def _log(
                 root_task_name = task_with_progress.task_name
                 progress_percent = task_with_progress.progress_percent
 
-                if progress_percent == "n/a":
-                    return
-
-                if not pbar:
-                    pbar = tqdm(total=100, unit="%", desc=root_task_name, maxinterval=self._LOG_POLLING_INTERVAL)
-
-                parsed_progress = float(progress_percent[:-1])
-                pbar.update(parsed_progress - pbar.n)
+                has_relative_progress = progress_percent != "n/a"
+                if pbar is None:
+                    if has_relative_progress:
+                        pbar = tqdm(total=100, unit="%", desc=root_task_name, maxinterval=self._LOG_POLLING_INTERVAL)
+                    else:
+                        # TODO add {n_fmt} once task_with_progress provides the absolute progress
+                        pbar = tqdm(
+                            total=None,
+                            unit="",
+                            desc=root_task_name,
+                            maxinterval=self._LOG_POLLING_INTERVAL,
+                            bar_format="{desc} [elapsed: {elapsed} {postfix}]",
```

Comment:
This is the relevant change I guess, showing elapsed time when no progress can be computed?

Reply:
yes correct

Reply:
and the `pbar.refresh` to actually update the elapsed time shown to the user